README.md

```diff
@@ -1 +1,9 @@
 # cs-5666-scatt-gp
+
+## Demo Instructions
+* Login to the Appalachian student server.
+* Clone the SCATT repo with the command `git clone https://github.com/JaysGitLab/cs-5666-scatt-gp.git`.
+* Go into the SCATT directory with the command `cd cs-5666-scatt-gp`.
+* Enter `make scatt` and hit return.
+* Enter the path to your SCATT submissions directory when prompted.
+* View the report output.
```

Comment:
@haietza @KaraBeason This is probably going to be too high level for now, but should we mention what the report is going to say right now, or separate it's view on screen better? Because right now after you enter the folder path and hit enter it just says Report Pass. I knew what it was going to say and it took me a sec to realize that it was done, so we might want to mention what it does so they aren't waiting for it to do something else or wonder if that's what it was suppose to do. What do you guys think?